libs/api/domains/intellectual-properties/src/lib/models/person.model.ts

```diff
@@ -0,0 +1,41 @@
+import { ObjectType, Field } from '@nestjs/graphql'
+import { Country } from './country.model'
+
+@ObjectType('IntellectualPropertiesDesignPerson')
+export class Person {
+  @Field(() => String, { nullable: true })
+  id?: string
+
+  @Field(() => String, { nullable: true })
+  name?: string
+
+  @Field(() => String, { nullable: true })
+  address?: string
+
+  @Field(() => String, { nullable: true })
+  postalCode?: string
+
+  @Field(() => String, { nullable: true })
+  city?: string
+
+  @Field(() => String, { nullable: true })
+  county?: string
+
+  @Field(() => Country, { nullable: true })
+  country?: Country
+
+  @Field(() => String, { nullable: true })
+  email?: string
+
+  @Field(() => String, { nullable: true })
+  telephone?: string
+
+  @Field(() => String, { nullable: true })
+  mobilePhone?: string
+
+  @Field({ nullable: true })
+  isForeign?: boolean
+
+  @Field(() => String, { nullable: true })
+  ssn?: string
```

Comment:
~ The monorepo favors `nationalId` over `ssn`.

Reply:
ah oops, will fix